src/components/ResultsMap.vue

```diff
@@ -29,11 +34,12 @@ export default {
     return {
 			markers: this.searchResults,
 			centerMap: {
-				lat: 40, lng: -105.0465427
+				lat: this.$store.state.geolocation.coords.latitude, 
+				lng: this.$store.state.geolocation.coords.longitude
 			},
-			selectedMarker: {}
-    }
-  },
+			selectedMarker: ""
```

Comment:
I had to set `selectedMarker`'s default value to an empty string instead of an empty object so that the new green marker could be conditionally rendered based on if `selectedMarker` is truthy